src/Components/Login/SignIn.js

```diff
@@ -0,0 +1,120 @@
+import { faEye } from "@fortawesome/free-solid-svg-icons";
```

Comment:
The folder name can be changed.

`Login` to `Auth`  because you have both the files in the `Login` folder `SignIn` and `SignUp` so changing it to `Auth` would make sense just a suggestion.